scipy/integrate/_quad_vec.py

```diff
@@ -144,36 +155,41 @@ def quad_vec(f, a, b, epsabs=1e-200, epsrel=1e-8, norm='2', cache_size=100e6, li
         Default: 'gk21' for finite intervals and 'gk15' for (semi-)infinite
     full_output : bool, optional
         Return an additional ``info`` dictionary.
+        
+        .. deprecated:: 1.11.0
+           Parameter `full_output` is deprecated and will be removed in version
+           1.13.0. When `full_output` is unspecified, the contents of ``info``
+           are provided as attributes of the result object.
     args : tuple, optional
         Extra arguments to pass to function, if any.
 
         .. versionadded:: 1.8.0
 
     Returns
     -------
-    res : {float, array-like}
-        Estimate for the result
-    err : float
-        Error estimate for the result in the given norm
-    info : dict
-        Returned only when ``full_output=True``.
-        Info dictionary. Is an object with the attributes:
-
-            success : bool
-                Whether integration reached target precision.
-            status : int
-                Indicator for convergence, success (0),
-                failure (1), and failure due to rounding error (2).
-            neval : int
-                Number of function evaluations.
-            intervals : ndarray, shape (num_intervals, 2)
-                Start and end points of subdivision intervals.
-            integrals : ndarray, shape (num_intervals, ...)
-                Integral for each interval.
-                Note that at most ``cache_size`` values are recorded,
-                and the array may contains *nan* for missing items.
-            errors : ndarray, shape (num_intervals,)
-                Estimated integration error for each interval.
+    A result object with the following attributes:
+
+        integral : {float, array-like}
+            Estimate for integration
+        abserr : float
+            Error estimate for the result in the given norm
```

Comment:
```suggestion
        integral : {float, array-like}
            Estimate of the integral.
        abserr : float
            Norm of the individual integral error estimates (as specified by
            `norm`).
```
We have the chance now to reconsider whether we should return each of the errors individually. Same goes for `success`, `status`, maybe `neval`, etc. Personally, I think that if we're going to vectorize functions like this, they should provide essentially the same information as if we were looping over the scalar version. For example, see what I'm proposing for `chandrupatla` (a scalar root finder) in https://github.com/scipy/scipy/issues/7242#issuecomment-1370131973.

Reply:
> We have the chance now to reconsider whether we should return each of the errors individually.

Is this not what `errors` contains?

Looking at the code I think it would be feasible for the others. I have not used this function in anger so don't have a strong opinion on how useful this would be.

Reply:
Oops, I didn't check. It's documented as "`float`".

Reply:
You're right I am confusing the intervals vs components of the vectorised function

Reply:
I had another look over the code; the part where the norm of taken is about as deep into the internals of the code as you can get. Whilst it probably would be possible to do what your suggesting it would be a non-trivial change so perhaps beyond the scope of this pr?

Reply:
Ok, no problem! We'll leave it alone.